integration_tests/src/main/python/parquet_test.py

```diff
@@ -849,7 +848,7 @@ def test_parquet_read_nano_as_longs_true(std_input_path):
         conf=conf)
 
 
-@disable_ansi_mode  # https://github.com/NVIDIA/spark-rapids/issues/5114
+@allow_non_gpu('ProjectExec') # ProjectExec will fallback to CPU on Spark-4.0 due to some optimizations.
```

Comment:
Could you explain this a little more? The test is named `test_many_column_project` and we are falling back on the project, so it feels odd that we think it is okay and we don't have a follow on issue to fix it.

Reply:
Thanks for the suggestion. Kept the test as is and mentioned the multiply issue in the comment. PTAL.